feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -100,6 +100,9 @@ class PreviewViewModel @Inject constructor(
                 cancel()
             }
         }
+        viewModelScope.launch {
```

Comment:
I think we should handle this from inside the `CameraUseCase` without needing the ViewModel's manual intervention.

When `runningCameraJob.cancel()` is called I think we'll get to `awaitCancellation()` in CameraXCameraUseCase.

From there we can do a finally block, and call `cameraProvider.unbindAll()`

Lets wait to hear from @temcguir on this too.